seacrowd/sea_datasets/wikiann/wikiann.py

```diff
@@ -71,15 +68,20 @@
 Minangkabau	min	min
 Sundanese	su	sun
 Acehnese	ace	ace
-Malay	ms	mly
+Malay	ms	zlm
 Banyumasan	map-bms	map-bms
+Myanmar my mya
+Tagalog tl tgl
+Thailand th tha
+Vietnam vi vie
+Khmer km khm
 
 
 """
 
 _HOMEPAGE = "https://github.com/afshinrahimi/mmner"
 
-_LICENSE = "Apache-2.0 license"
+_LICENSE = Licenses.APACHE_2_0.value
 
 _URLs = {
     "wikiann": "https://s3.amazonaws.com/datasets.huggingface.co/wikiann/1.1.0/panx_dataset.zip",
```

Comment:
consider removing this to reduce _URLs complexity
```suggestion
_URLs = {
    "https://s3.amazonaws.com/datasets.huggingface.co/wikiann/1.1.0/panx_dataset.zip",
```

and pls adjust it too where the `_URLs` var is being used

Reply:
adjusted